core/src/test/java/org/apache/iceberg/TableMetadataParserTest.java

```diff
@@ -37,30 +40,24 @@
 import org.apache.iceberg.io.OutputFile;
 import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 import org.apache.iceberg.types.Types.BooleanType;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
 
-@RunWith(Parameterized.class)
+@ExtendWith(ParameterizedTestExtension.class)
 public class TableMetadataParserTest {
 
   private static final Schema SCHEMA = new Schema(optional(1, "b", BooleanType.get()));
 
-  @Parameterized.Parameters(name = "codecName = {0}")
-  public static Object[] parameters() {
-    return new Object[] {"none", "gzip"};
+  @Parameters(name = "codecName = {0}")
+  protected static List<Object> parameters() {
```

Comment:
given that this test is independent from `TestBase`, we can probably make this private

Reply:
Yes sure, thanks for the advice.